<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PilotDetails</class>
 <widget class="QDialog" name="PilotDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>615</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AUA325</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Pilot Information</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="lblPilotInfo">
        <property name="text">
         <string>&lt;strong&gt;PILOT: SFC Donald Fauntleroy Duck&lt;/strong&gt;&lt;br/&gt;Home: Rome, IT&lt;br/&gt;On IVANDE for 01:32, IvAp 1.3.4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Aircraft Information</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="lblAircraftInfo">
        <property name="text">
         <string>Aircraft: &lt;strong&gt;A319&lt;/strong&gt;&lt;br/&gt;Altitude: &lt;strong&gt;33052&lt;/strong&gt; feet&lt;br/&gt;Ground Speed: &lt;strong&gt;542&lt;/strong&gt; kts</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Flight Plan</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;strong&gt;Flight Status:&lt;/strong&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblFlightStatus">
          <property name="text">
           <string>En route (35%), ETA 02:33</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>FROM:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buttonDest">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>LOWI (Innsbruck, AT)</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="buttonFrom">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>LOWW (Vienna, AT)</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>DEST:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Route:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblRoute">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>KAPPI UW705 BEROK UW653 TULSI</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblRemarks">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;small&gt;Beginner Pilot&lt;/small&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonShowOnMap">
       <property name="text">
        <string>&amp;Show on Map</string>
       </property>
       <property name="shortcut">
        <string>S</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddFriend">
       <property name="text">
        <string>Add Friend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonFrom</tabstop>
  <tabstop>buttonDest</tabstop>
  <tabstop>buttonShowOnMap</tabstop>
  <tabstop>buttonAddFriend</tabstop>
  <tabstop>buttonClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>PilotDetails</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonShowOnMap</sender>
   <signal>clicked()</signal>
   <receiver>PilotDetails</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
